src/auth/auth.go

```diff
@@ -47,12 +47,6 @@ const (
 	// be removed from the set of cluster super-admins.
 	RootUser = "pach:root"
 
-	// PpsUser is a special, unrevokable cluster administrator account used by PPS
-	// to create pipeline tokens, close commits, and do other necessary PPS work.
-	// It's not possible to authenticate as ppsUser (pps reads the auth token for
-	// this user directly from etcd).
-	PpsUser = `pach:pps`
-
```

Comment:
My understanding is we wanted to move away from this, but I haven't kept up to date

Reply:
Correct. There are three situations where we use the pps superuser today:
- When a user makes an RPC, we use the superuser to write to the spec repo - this should fix that case.
- Background tasks in pachd use the super-user token because there's no user identity. They can use internal APIs or use the pipeline's token from etcd.
- Pipelines making calls to the sidecar use the pipeline's token.